assets/js/directives/buy-quick-start.directive.js

```diff
@@ -39,18 +43,14 @@ function buyQuickStart (currency, buySell, Alerts, $interval) {
 
     let fetchingQuote;
     let startFetchingQuote = () => {
-      fetchingQuote = $interval(() => scope.getQuote(), 1000 * 60);
+      fetchingQuote = $interval(() => scope.getExchangeRate(), 1000 * 60);
```

Comment:
If you use a timeout instead of an interval, might not need the stopFetching / startFetching logic.